schemas/variant_calling_schema.json

```diff
@@ -3,12 +3,82 @@
   "schema": {
```

Comment:
Just a general question to refresh my memory, where/how do we supply the Project ID?  I don't see it in the schema

Reply:
Not critical, just a small cleanup thing, we could remove the references to `exposure` and `family_history` in `donor` since those entities won't be used in the EUCANCan model

Reply:
> Just a general question to refresh my memory, where/how do we supply the Project ID? I don't see it in the schema

project_id or study_id are included in the base schema, so is not defined here in Dynamic ones.

Reply:
> Not critical, just a small cleanup thing, we could remove the references to `exposure` and `family_history` in `donor` since those entities won't be used in the EUCANCan model

`exposure` and `family_history` are not defined in the schema, but they are listed in the property enum. we can remove them completely.